plugins/LocalFileOutputDevice/LocalFileOutputDevice.py

```diff
@@ -24,8 +24,8 @@
 class LocalFileOutputDevice(ProjectOutputDevice):
     """Implements an OutputDevice that supports saving to arbitrary local files."""
 
-    def __init__(self, parent = None):
-        super().__init__(device_id = "local_file", parent = parent)
+    def __init__(self, add_to_output_devices: bool = True, parent = None):
```

Comment:
Other plugins (not ours) that implement ProjectOutputDevice will have to do this as well, then? It's an API break in any case.

Not necessarily a deal-breaker, but we should be careful. Let's ask peoples opinions on this during eCCB.

Reply:
Never mind, I was made aware that it has a default. (which should have been clear from the default being right in my face in the first place :-p )